packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -404,6 +416,35 @@ class Doctor {
           break;
       }
       if (sendEvent) {
+        if (validator is GroupedValidator) {
+          // Ensure that all of the subvalidators in the group have
+          // a corresponding subresult incase a validator crashed
+          if (validator.subValidators.length != validator.subResults.length) {
+            continue;
```

Comment:
If you continue here, it won't print out any details about the validator when you do `flutter doctor`, is that intended?

Reply:
Glad you caught that! I just noticed that too while trying to debug the google3 errors. But yes, the continue should be moved inside the loop i created, I'm thinking using a `try` and catch any `RangeError` that comes out and continue there instead

Reply:
Just fixed it, lmk if that looks good to you

Reply:
Code looks good, will you add a test for it, though?

Reply:
Just added now in now! :)